edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc

```diff
@@ -1,8 +1,8 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="cnf-image-based-upgrade-for-sno"]
-= Performing an image-based upgrade for {sno} clusters
+[id="cnf-image-based-upgrade"]
+= Performing an image-based upgrade for {sno} clusters with {lcao}
```

Comment:
At the risk of introducing a suggestion that would require you to make several updates throughout this PR, would the title make more sense if it said either "with the {lcao}" or "using the {lcao}"? Up to you, as of now it does sound a little awkward imo